XenModel/FriendlyNames.resx

```diff
@@ -1856,6 +1856,9 @@
   <data name="Message.name-wlb_vm_relocation" xml:space="preserve">
     <value>WLB VM Relocation</value>
   </data>
+  <data name="OPERATIONAL_METRICS_HOSTLOAD" xml:space="preserve">
+    <value>CPU Load</value>

```

Comment:
Looks this will show on UI. So I suggest to use "Host Load" or "Host CPU Load" that keeps same style to `Control Domain Load` for `loadavg`

Reply:
Yeah, I was wondering about that, then I thought I'd drop the "Host" bit because we don't use it in the other host CPU datasources, but I'll add it back. Thanks.

Reply:
Since it's similar to `loadavg` I also changed the category type shown by the UI.